Kernel/CMakeLists.txt

```diff
@@ -205,6 +205,8 @@ set(KERNEL_SOURCES
     FileSystem/SysFS/Subsystems/Kernel/Variables/CoredumpDirectory.cpp
     FileSystem/SysFS/Subsystems/Kernel/Variables/Directory.cpp
     FileSystem/SysFS/Subsystems/Kernel/Variables/DumpKmallocStack.cpp
+    FileSystem/SysFS/Subsystems/Kernel/Variables/Hostname.cpp
+    FileSystem/SysFS/Subsystems/Kernel/Variables/FixedStringBufferVariable.cpp
```

Comment:
I know this list leaves a lot to be desired, but can you add them in alphabetical order?